assets/src/edit-story/components/panels/sizePosition.js

```diff
@@ -238,6 +265,28 @@ function SizePositionPanel({
           </Button>
         </Row>
       )}
+      {/** X/Y */}
+      <Row expand>
+        <BoxedNumeric
+          suffix={_x('X', 'Position on X axis', 'web-stories')}
+          value={x}
+          min={MIN_MAX.X.MIN - width}
+          max={MIN_MAX.X.MAX}
+          onChange={(value) => pushUpdate({ x: value })}
+          aria-label={__('X position', 'web-stories')}
+          canBeNegative
+        />
+        <Spacer />
+        <BoxedNumeric
+          suffix={_x('Y', 'Position on Y axis', 'web-stories')}
+          value={y}
+          min={MIN_MAX.Y.MIN - height}
```

Comment:
This is connected to @miina comment about `0-safezone`, I think we can leave `Y.MIN=0` but here we need to account for the safezone (box height is irrelevant).

Editor:
![image](https://user-images.githubusercontent.com/21036224/90872450-f0a57500-e39c-11ea-9094-d186d63d2730.png)

Output:
![image](https://user-images.githubusercontent.com/21036224/90872512-0b77e980-e39d-11ea-9e2c-630b9704e71e.png)